projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -563,8 +571,22 @@ export class IgxDatePickerComponent implements IDatePicker, ControlValueAccessor
      *<igx-date-picker (onClose)="close($event)" cancelButtonLabel="cancel" todayButtonLabel="today"></igx-date-picker>
      *```
      */
+    @DeprecateProperty(`'onClose' @Ouput property is deprecated\nUse 'onClosed' instead.`)
     @Output()
     public onClose = new EventEmitter<IgxDatePickerComponent>();
+
+    /**
+     *An event that is emitted when the `IgxDatePickerComponent` is closed.
```

Comment:
Maybe this is more understandable `An event that is emitted **after** the `IgxDatePickerComponent` is closed.`